client/my-sites/checkout/checkout/checkout-terms.jsx

```diff
@@ -19,7 +19,11 @@ class CheckoutTerms extends React.Component {
 		return (
 			<Fragment>
 				<div className="checkout__terms">
-					<strong>By checking out:</strong>
+					<strong>
+						{ this.props.translate( 'By checking out:', {
+							comment: 'Headline before a list of terms the customer is agreeing to on checkout.',
```

Comment:
I think we should add a link to this screenshot to make it more clear:  

![](https://user-images.githubusercontent.com/1379730/55166390-66c09080-5145-11e9-9c13-6c1b3b693786.png)

Reply:
I was thinking of doing that, but somewhat afraid this image url may go bad at some point. But I'll add it.